tools/syz-repro/repro.go

```diff
@@ -22,6 +22,8 @@ var (
 	flagConfig = flag.String("config", "", "manager configuration file (manager.cfg)")
 	flagCount  = flag.Int("count", 0, "number of VMs to use (overrides config count param)")
 	flagDebug  = flag.Bool("debug", false, "print debug output")
+	flagOutput = flag.String("output", "./output.txt", "output description file (output.txt)")
```

Comment:
`/` is not the path separator on all OSes. For example, Windows uses `\`. There are functions to form portable paths in `path/filepath` package.

Reply:
fixed